myceliald/src/main.rs

```diff
@@ -64,6 +63,7 @@ async fn run() -> Result<()> {
 
 #[tokio::main]
 async fn main() {
+    tracing_subscriber::fmt::init();
```

Comment:
tracing?

Reply:
incoming
but if serious - forgot to drop it after checks

Reply:
👍